test/specs/browser/modal_test.js

```diff
@@ -263,4 +263,253 @@ describe('modals', function desc() {
             });
         });
     });
+
+    describe('EditServerModal', () => {
+        let editServerView;
+
+        beforeEach(async () => {
+            const mainView = this.app.windows().find((window) => window.url().includes('index'));
+            const dropdownView = this.app.windows().find((window) => window.url().includes('dropdown'));
+            await mainView.click('.TeamDropdownButton');
+            await dropdownView.hover('.TeamDropdown .TeamDropdown__button:nth-child(1)');
+            await dropdownView.click('.TeamDropdown .TeamDropdown__button:nth-child(1) button.TeamDropdown__button-edit');
+
+            editServerView = await this.app.waitForEvent('window', {
+                predicate: (window) => window.url().includes('editServer'),
+            });
+        });
+
+        it('should not edit team when Cancel is pressed', async () => {
+            await editServerView.click('#cancelNewServerModal');
```

Comment:
Sorry I missed this earlier, but shouldn't the test edit the modal before clicking cancel so it doesn't apply the changes?